src/database.ts

```diff
@@ -337,6 +337,7 @@ class Database extends common.GrpcServiceObject {
   databaseDialect?: EnumKey<
     typeof databaseAdmin.spanner.admin.database.v1.DatabaseDialect
   > | null;
+  observabilityConfig: ObservabilityOptions | undefined;
```

Comment:
should use underscore to refer this property as private. Else customer can override the property at database level . Or is there any other way we can avoid this ?

Reply:
I suffixed it with an underscore so `observabilityConfig_`.